websubhub-examples/kafka-hub/hub/hub_service.bal

```diff
@@ -40,7 +40,7 @@ service object {
     isolated remote function onRegisterTopic(websubhub:TopicRegistration message, http:Headers headers)
                                 returns websubhub:TopicRegistrationSuccess|websubhub:TopicRegistrationError|error {
         if config:SECURITY_ON {
-            check security:authorize(headers, ["register_topic"]);
+            check security:authorizeSubscriber(headers, message.topic);
```

Comment:
Should not this be `security:authorizePublisher(headers, message.topic)` ?

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-websubhub/pull/236/commits/c11d3faf75b7f8021b70197983a1dc38b64a36b4